src/libraries/System.Runtime.Extensions/tests/System/Convert.ToHexString.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Text;
+using Xunit;
+
+namespace System.Tests
+{ 
+    public class ConvertToHexStringTests
+    {
+        [Fact]
+        public static void KnownByteSequence()
+        {
+            byte[] inputBytes = new byte[] { 0x00, 0x01, 0x02, 0xFD, 0xFE, 0xFF };
+            Assert.Equal("000102FDFEFF", Convert.ToHexString(inputBytes));
+        }
+
+        [Fact]
+        public static void CompleteValueRange()
+        {
+            byte[] values = new byte[256];
+            StringBuilder sb = new StringBuilder(256);
+            for (int i = 0; i < values.Length; i++)
+            {
+                values[i] = (byte)i;
+                sb.Append(i.ToString("X2"));
+            }
+
+            Assert.Equal(sb.ToString(), Convert.ToHexString(values));
+        }
+
+        [Fact]
+        public static void ZeroLength()
+        {
+            byte[] inputBytes = Convert.FromHexString("000102FDFEFF");
+            Assert.Same(string.Empty, Convert.ToHexString(inputBytes, 0, 0));
+        }
+
+        [Fact]
+        public static void InvalidInputBuffer()
+        {
+            Assert.Throws<ArgumentNullException>(() => Convert.ToHexString(null));
```

Comment:
Nit: For all of these argument validations, can you use `AssertExtensions.Throws` and pass the expected argument name as the first argument?

Reply:
Done